### ISSUE TITLE
Custom aria-label on indicators

### ISSUE THREAD

**Issue by aygee** [Learus/react-material-ui-carousel#105]
Hi, is there a way to pass in the aria-label to the indicators? They need to be different per indicator, eg "Page [number]"

**Comment by benhammondmusic** [comment]
Or even a way to add aria-hidden ? Right now the screen reader just reads generic "button" for each of them... is there a workaround for this?

**Comment by Learus** [comment]
Hello there.

I'm currently implementing a solution for this issue. The plan is as follows. I will extend the typings of the props given to the indicators and navigation buttons to allow for all aria properties.

Moreover, I'd like to add `aria-label`s to those elements out of the box, but I am uncertain as to what exactly those `aria-label`s should be. I am not that familiar with the concept sadly, and I'd like more a more expert opinion if possible.

Any suggestions? @benhammondmusic

**Comment by Learus** [comment]
I've extended the props customizing indicators and buttons to support all aria properties. I also added default `aria-label`s to the indicators: `"carousel indicator {i}"` where `i` is the indicator index + 1. Will change if there is a better suggestion, given the functionality.

Note that any `aria-label` that you add to the indicators using the (`active`)`indicatorIconButtonProps` will be rendered with the indicator index next to it. 
For example, if you specify:
```jsx
//...Carousel...
indicatorIconButtonProps={{
    'aria-label': 'MyIndicatorLabel'
}}
//...
```
the rendered label for the first indicator will be `"MyIndicatorLabel 1"` and so on.

Affected props are:
- `navButtonsWrapperProps`
- `navButtonsProps`
- `indicatorIconButtonProps`
- `activeIndicatorIconButtonProps`
- `indicatorContainerProps`


Please upgrade to `v2.3.0` or over.

**Comment by aygee** [comment]
This is great thank you @Learus. It will be better if we can pass in an aria label for each of the indicator so that it would work with languages that don't use numerical characters for the numbers.

**Comment by benhammondmusic** [comment]
@Learus thank you very much for this; sorry it took me so long to get around to checking out your update. I appreciate the work!
